tests/test_core.py

```diff
@@ -1915,6 +1945,8 @@ def test_stack_void(self):
 
     self.do_run_from_file(src, output)
 
+  # Broken for binaryen2
```

Comment:
i believe it's not broken, just extremely slow in the binaryen interpreter. but that was a while ago - what type of breakage did you see?